pkg/cloud/azureprovider.go

```diff
@@ -1379,8 +1382,25 @@ func (az *Azure) ServiceAccountStatus() *ServiceAccountStatus {
 	}
 }
 
+const rateCardPricingSource = "Rate Card API"
+
+// PricingSourceStatus returns the status of the rate card api
 func (az *Azure) PricingSourceStatus() map[string]*PricingSource {
-	return make(map[string]*PricingSource)
+	sources := make(map[string]*PricingSource)
+	rcps := &PricingSource {
+		Name: rateCardPricingSource,
+		Error: az.RateCardPricingStatus,
```

Comment:
Does this imply that `RateCardPricingStatus` should really be of type `error`? The only place it seems to be used is here to convey error information.

Reply:
It is only used for that, I was following a president set in AWS implementation, but they should probably change